templates/monitoring/document_sample_nonprod_v1/query.sql

```diff
@@ -0,0 +1,69 @@
+WITH extract_decoded AS (
+  SELECT
+    metadata.document_namespace,
+    metadata.document_type,
+    metadata.document_version,
+    '' AS error_message,
+    payload
+  FROM
+    `moz-fx-data-shar-nonprod-efed.payload_bytes_decoded.*`
+  WHERE
+    submission_timestamp
+> TIMESTAMP_SUB(current_timestamp, INTERVAL 1 day)
+),
+extract_error AS (
+  SELECT
+    document_namespace,
+    document_type,
+    document_version,
+    error_message,
+    payload
+  FROM
+    `moz-fx-data-shar-nonprod-efed.payload_bytes_error.*`
+  WHERE
+    submission_timestamp
+> TIMESTAMP_SUB(current_timestamp, INTERVAL 1 day)
+AND error_type = 'ParsePayload'
+)
+SELECT
+  n_samples,
+  document_is_decoded,
+  sample.*
+FROM
+  (
```

Comment:
Can we make this another named CTE rather than a subquery? I think it would be easier to follow the logic by not having this nested here.